apps/infrastructure/src/iam_policy.tf

```diff
@@ -100,7 +85,6 @@ data "aws_iam_policy_document" "ecs_task_role_s3" {
       "s3:PutObjectAcl"
     ]
     resources = concat(
-      [aws_s3_bucket.cms_medialibrary_bucket.arn],
       [for name, bucket in aws_s3_bucket.cms_multitenant_medialibrary_bucket : bucket.arn]
     )
```

Comment:
The concat can now be removed by only leaving the list comprehension at line 88

Reply:
concat removed with last commit